evap/staff/views.py

```diff
@@ -58,8 +58,6 @@ def get_courses_with_prefetched_data(semester):
             "degrees"
         ).annotate(
             num_contributors=Count("contributions", filter=~Q(contributions__contributor=None), distinct=True),
-            num_textanswers=Count("contributions__textanswer_set", distinct=True),
-            num_reviewed_textanswers=Count("contributions__textanswer_set", filter=~Q(contributions__textanswer_set__state=TextAnswer.NOT_REVIEWED), distinct=True),
```

Comment:
any ideas about this or is it okay to remove the annotations? `num_textanswers` does now return `0` if `can_publish_text_results` is `False`, but this counted the data directly.

Reply:
It will trigger n queries for n courses in the semester right now, since in semester_view we make the sum of all num_textanswers. I guess this could be reduced to one query there (?). It's the same for num_reviewed_textanswers.